src/main/java/wolox/training/exceptions/NotFound.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class NotFound extends ResponseStatusException  {
```

Comment:
Always finish the class with `Exception` in this case could be `BookNotFoundException `